modules/disabling-insights-operator-alerts.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: CONCEPT
+[id="disabling-insights-operator-alerts_{context}"]
+= Disabling Insights Operator alerts
+
+You can stop Insights Operator from firing alerts to the cluster prometheus instance.
+
+. Navigate to *Workloads* -> *Secrets*.
+. Click *Show default projects*.
+. Select the *openshift-config* project.
+. Search for the *support* secret using the *Search by name* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
+. Click the *Options* menu {kebab}, and then click *Edit Secret*.
+. Click *Add Key/Value*.
+. Enter `disableInsightsAlerts` as the key with the value `True`, and click *Save*.
+
+After you save the changes, Insights Operator will no longer send alerts to the cluster prometheus instance. 
```

Comment:
I think Prometheus should be with capital P. Perhaps I would add that it's not applied immediately, but takes few minutes. WDYT?